website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -380,6 +380,95 @@ To create a column that has right-aligned content, set `@align` to `right` on bo
 </Hds::Table>
 ```
 
+### Scrollable table
+
+As a general principle, tables should contain as little information as possible. Consuming a large amount of data in a tabular format leads to an intense cognitive load for the user. We recommend using functionalities like [pagination](/components/pagination), [sorting](/components/table?tab=code#sortable-table), and [filtering](/patterns/filter-patterns) to reduce this load.
+
+That said, there may be cases in which it's necessary to show a table with a large number of columns, and allow the user to scroll horizontally. In this case the consumer can use different approaches, depending on their context, needs and design specs.
+
+Below we show a couple of examples of how a scrollable table could be implemented: use them as starting point (your mileage may vary).
+
+#### Using a container with `overflow: auto`
+
+In most cases, wrapping the table with a container that has `overflow: auto` does the trick.
+
+The table layout is `auto` (default) which means the browser will try to optimize the width of the columns to fit their different content. In some cases, this will mean the content may wrap (see the `Phone` column as an example) in which case you may want to apply a `width` to suggest to the browser to apply a specific width to a column (see the `Biography` column).
```

Comment:
```suggestion
The default table layout is `auto` which means the browser will try to optimize the width of the columns to fit their different content. In some cases, this will mean the content may wrap (see the `Phone` column as an example) in which case you may want to apply a `width` to apply a specific width to a column (see the `Biography` column).
```

Reply:
@jorytindall I am going to leave `to suggest to the browser` because technically that's what's happening (hence this ticket): the browser has its own heuristics to determine if apply that width or not, depending on the content on the table. I've added a link to this page/content https://www.w3.org/TR/WD-CSS2-971104/tables.html#h-17.2 so maybe it's clearer what I mean